pkg/image/api/v1/trigger/trigger.go

```diff
@@ -0,0 +1,24 @@
+package trigger
+
+// ObjectFieldTrigger links a field on the current object to another object for mutation.
+type ObjectFieldTrigger struct {
```

Comment:
One of my complaints with the original trigger object which is inlined in DCs is that it's impossible to track individual triggers (what happens if users reorder them and update them at the same time?). How about using a UID?